src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -386,6 +449,46 @@ public JSONObject getDebugState()
         return debugState;
     }
 
+    public void setVideoConstraints(Map<String, VideoConstraints> newVideoConstraintsMap)
+    {
+        Map<String, VideoConstraints>
+            videoConstraintsMapCopy = this.videoConstraintsMap;
```

Comment:
The purpose isn't to create a deep copy, it's to prevent the reference from changing while we're accessing it (we're doing `VideoConstraints oldVideoConstraints = videoConstraintsMapCopy.get(endpointId);` in this method.